.github/workflows/run-tests.yml

```
@@ -0,0 +1,238 @@
+name: Run tests
+
+on: [pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.1
+          tools: composer
+        env:
+          fail-fast: true
+
+      - name: Software versions
+        id: versions
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            php --version && composer --version
+
+      - name: Checkout Code
+        id: checkout
+        uses: actions/checkout@v3
```

I have a shiny new thing for that PHP version.

```suggestion
      - name: Checkout Code
        id: checkout
        uses: actions/checkout@v3
        
      - name: Extract PHP Version
        id: php
        uses: docker://ghcr.io/un-ocha/actions:extract-php-version-main
        with:
          docker_file: 'docker/Dockerfile'
          docker_image: 'public.ecr.aws/unocha/php-k8s'
          
      - name: Setup PHP with PECL extension
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ steps.php.outputs.php_version }}
          tools: composer
        env:
          fail-fast: true

      - name: Software versions
        id: versions
        uses: cafuego/command-output@main
        with:
          run: |
            php --version && composer --version


```